embedded-postgresql/src/main/java/com/playtika/test/postgresql/EmbeddedPostgreSQLBootstrapConfiguration.java

```diff
@@ -54,7 +55,8 @@ public ConcretePostgreSQLContainer postgresql(ConfigurableEnvironment environmen
         log.info("Starting postgresql server. Docker image: {}", properties.dockerImage);
 
         ConcretePostgreSQLContainer postgresql =
-                new ConcretePostgreSQLContainer(properties.dockerImage)
+                new ConcretePostgreSQLContainer(DockerImageName.parse(properties.dockerImage)
+                        .asCompatibleSubstituteFor("postgres"))
```

Comment:
curious, what is this change about

Reply:
Here is an example for elasticseacrh:
Caused by: org.springframework.beans.factory.BeanCreationException: 
Error creating bean with name 'embeddedElasticSearch' defined in class path resource [com/playtika/test/elasticsearch/EmbeddedElasticSearchBootstrapConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.testcontainers.elasticsearch.ElasticsearchContainer]: Factory method 'elasticSearch' threw exception; nested exception is java.lang.IllegalStateException: Failed to verify that image 'elasticsearch:7.7.0' is a compatible substitute for 'docker.elastic.co/elasticsearch/elasticsearch'. This generally means that you are trying to use an image that Testcontainers has not been designed to use. If this is deliberate, and if you are confident that the image is compatible, you should declare compatibility in code using the `asCompatibleSubstituteFor` method. For example:
   DockerImageName myImage = DockerImageName.parse("elasticsearch:7.7.0").asCompatibleSubstituteFor("docker.elastic.co/elasticsearch/elasticsearch");